src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -109,7 +109,10 @@ public void periodic() {
 		}
 
 		if (m_feederTimer.get() > 0) {
-			m_topFeeder.set(ShooterConstants.kTopFeederSpeedFast);
+			if (m_bottomShooterPID.atSetpoint() && m_topShooterPID.atSetpoint()) {
+				m_topFeeder.set(ShooterConstants.kTopFeederSpeedFast);
+			}
+
```

Comment:
I assume this works but it is not implemented well. At this point if it works it's ok to merge.

Reply:
It works, I can't really think of another way to implement it at the moment and we just need it working for competition.